templates/vmclasses.yaml

```diff
@@ -0,0 +1,103 @@
+---
```

Comment:
we already have vmclasses [manifests](https://github.com/deckhouse/virtualization/blob/main/templates/virtualization-controller/vmclasses-default.yaml)

do we need to delete old manifests?

Reply:
oops